common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java

```diff
@@ -780,6 +789,14 @@ private static void writeUnparsedType(String unparsedType, JsonGenerator gen) th
     gen.writeEndObject();
   }
 
+  private static void writeExternalType(Types.ExternalType externalType, JsonGenerator gen)
+      throws IOException {
+    gen.writeStartObject();
+    gen.writeStringField(TYPE, EXTERNAL);
+    gen.writeStringField(CATALOG_STRING, externalType.catalogString());
```

Comment:
Do we need to handle a senario that catalog string is a complex string or json string?

Reply:
I think it's unnecessary because the external type only needs to pass the type string from the underlying catalog.

Reply:
What I mean is that if a `catalogString` itself passed in by user is a json string, will this serialization fail, do we need to escape?